.devcontainer/post_create.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+#
+# Other postCreateCommands that should only run in the Dev Container environment
+# Since these are called by the .devcontainer/devcontainer.json's `postCreateCommand`
+# they won't affect people building the software natively
+
+git config --global --add safe.directory /workspaces/prql
```

Comment:
> I observed the need for the git config ... --add safe.directory ... command in three environments: macOS 12.7.5 on Intel, macOS 14.5 on M2, and Win 11. So there's something about the DevContainer that triggers/requires it. This PR works around it cleanly.

Any thoughts on why this is needed in this repository and not in others?
Have you reported this issue upstream?

In my opinion, this is a problem that should be resolved upstream, and doing something about it here is the wrong direction.